### ISSUE TITLE
Python 3 idle3 not working

### ISSUE THREAD

**Issue by s0urce-c0de** [chromebrew/chromebrew#8282]
**Describe the bug**
idle 3 says:
```
** IDLE can't import Tkinter.
Your Python might not be configured for Tk. **
```
**To Reproduce**

1. I executed this command:

      ```shell
      idle3
      ```

      Output:

      ```
 ** IDLE can't import Tkinter.
Your Python might not be configured for Tk. **
      ```

**Expected behavior**
Open idle window

**Screenshots**

**Environment: Paste the output of `crew sysinfo -v` between the two `***` below**

***
<details><summary>Expand</summary>

 - Architecture: `x86_64` (`x86_64`)
 - Processor vendor: `GenuineIntel`
 - User space: `64-bit`
 - Chromebrew Kernel version: `5.15`
 - Chromebrew Running in Container: `false`

 - Chromebrew version: `1.33.9`
 - Chromebrew Prefix: `/usr/local`
 - Chromebrew libdir: `/usr/local/lib64`

 - Last update in local repository: c51b5b1 `Edge 113.0.1774.42-1 => 113.0.1774.50-1 (#8277) (8 hours ago)`

 - OS variant: `Chrome OS`
 - OS version: `hatch-release/R113-15436.0.0`
 - OS channel: `canary-channel`

</details>

</details>
***

**Additional context**
I am on canary channel, but I don't want to powerwash to reset, and I assume this is a python missing tkinter, so I posted a bug
What I think needs to be done:

      - Add python3-tk package, or ask user if they want it

**Comment by satmandu** [comment]
You can uncomment these lines in /usr/local/lib/crew/packages/python3.rb:
```
  # depends_on 'tcl' # Needed for tkinter support
  # depends_on 'tk'  # Needed for tkinter support
```
Then do a `crew reinstall -s python3` which should rebuild python with tk support.

**Comment by satmandu** [comment]
I'm adding tcl and tk to the builds for Python 3.11.4...

**Comment by s0urce-c0de** [comment]
ok, you might want to fix python2 too, but thanks!

**Comment by satmandu** [comment]
I don't think we are going to touch python2. It's no longer supported, and doesn't get updates.
